src/main/java/com/example/nasastealphoto/service/CameraService.java

```diff
@@ -0,0 +1,19 @@
+package com.example.nasastealphoto.service;
+
+import com.example.nasastealphoto.model.Camera;
+import com.example.nasastealphoto.repository.CameraRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class CameraService {
+
+    private final CameraRepository cameraRepository;
+
+    @Transactional
```

Comment:
I think that you can remove `@Transactional` form here. The Spring Data repository `save` is transactional by default
[javadoc](https://docs.spring.io/spring-data/data-jpa/docs/current/api/org/springframework/data/jpa/repository/support/SimpleJpaRepository.html)